input/start/openshift.md

```diff
@@ -1 +1,259 @@
-# Using Skupper with Red Hat OpenShift
+---
+title: Getting started
+---
+
+# Getting started with Skupper on OpenShift
```

Comment:
If this is a complement to the main getting started, then it's a weird one.  It's both OpenShift and site controller specific.  A natural complement would be something that is about using the site controller with any flavor of Kubernetes, like the CLI variant.